Sources/XMLCoder/Encoder/XMLEncoder.swift

```diff
@@ -796,7 +796,16 @@ fileprivate struct _XMLUnkeyedEncodingContainer : UnkeyedEncodingContainer {
     
     public mutating func encode<T : Encodable>(_ value: T) throws {
         self.encoder.codingPath.append(_XMLKey(index: self.count))
-        defer { self.encoder.codingPath.removeLast() }
+        print("unkeyed type: \(type(of: value))")
```

Comment:
I guess this was added for debugging purposes and could be removed before merge?